src/main/java/past/cwdt/bookstore/model/Book.java

```diff
@@ -26,8 +31,17 @@ public class Book {
     private String title;
     @Column(nullable = false)
     private String author;
+
+    @ManyToMany
```

Comment:
maybe it makes sense to add PERSIST cascade, in case you save a book with some new category

just a suggestion - decide whether it is suitable for you